modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SpringCodegen.java

```diff
@@ -126,7 +126,7 @@ public void processOpts() {
                         (sourceFolder + File.separator + configPackage).replace(".", java.io.File.separator), "HomeController.java"));
                 supportingFiles.add(new SupportingFile("swagger2SpringBoot.mustache",
                         (sourceFolder + File.separator + basePackage).replace(".", java.io.File.separator), "Swagger2SpringBoot.java"));
-                supportingFiles.add(new SupportingFile("application.properties",
+                supportingFiles.add(new SupportingFile("application.mustache",

```

Comment:
@vovan- can you please also do the same for `spring-mvc`?

https://github.com/swagger-api/swagger-codegen/pull/3307/files#diff-8642abd39c9b8b007a73f66dad7250caR139